examples/advanced_operations/add_dynamic_page_feed.py

```diff
@@ -141,17 +141,11 @@ def get_feed_details(client, customer_id, resource_name):
         A FeedDetails instance with information about the feed that was
         retrieved in the search request.
     """
-    query = """
-        SELECT
-            feed.attributes
-        FROM
-            feed
-        WHERE
-            feed.resource_name = "{}"
-        LIMIT 1
-    """.format(
-        resource_name
-    )
+    query = f"""
+        SELECT feed.attributes
+        FROM feed
+        WHERE feed.resource_name = '{resource_name}'
+        LIMIT 1"""
```

Comment:
not sure why we need a `LIMIT 1` given we are checking for a single resource. I believe the convention is to use quotes, not apostrophes, around literals, ie. `WHERE feed.resource_name = "{resource_name}"`

Reply:
Removed the limit.